docs/4.1/concept-basics.md

```diff
@@ -0,0 +1,40 @@
+# Teleport Basics
+
+This doc will introduce the basic concepts of teleport so you can get started managing access!
+
+First, get to know the components of Teleport. Teleport uses three different services which work together to make Teleport go: **Nodes**, **Auth**, and **Proxy**. 
+
+- **Teleport Nodes** are servers which can be accessed remotely via Teleport Auth. The Teleport Node service runs on a machine and is similar to the `sshd` daemon you may be familiar with. Users can log in to a Node with the `tsh ssh` command or via a web browser through the Teleport Proxy UI. 
```

Comment:
Here I would also note that users can use `ssh` on the client side because teleport implements `ssh` protocol, so people won't think that it's a separate custom protocol implementation.